@here/harp-utils/lib/GroupedPriorityList.ts

```diff
@@ -32,6 +32,15 @@ export class PriorityListGroup<T extends PriorityListElement> {
     clone(): PriorityListGroup<T> {
         return new PriorityListGroup<T>(this.priority, this.elements.slice());
     }
+
+    remove(element: T): boolean {
```

Comment:
please add documentation to the function

Reply:
done